docs/apis-tools/administration-sm-api/administration-sm-api-authentication.md

```diff
@@ -5,34 +5,61 @@ sidebar_position: 2
 description: "The Administration API for Self-Managed is a REST API and provides access to Console Self-Managed data. Requests and responses are in JSON notation."
 ---
 
-To authenticate for the API, generate a JWT token and pass it in each request:
+All Administration Self-Managed API requests require authentication. To authenticate, generate a [JSON Web Token (JWT)](https://jwt.io/introduction/) and pass it in each request.
+
+## Generating a token
```

Comment:
```suggestion
## Generate a token
```
I know we use gerunds (ing) but not sure we should? One to think about, perhaps not for this PR

Reply:
I'm going to hold off on this one for now.